package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@twilio-labs/plugin-rtc",
-  "version": "0.1.7",
+  "version": "0.3.0",
```

Comment:
Oh weird. I think we need to submit a PR to the current master branch that updates the version to 0.2.0